authenticate/state.go

```diff
@@ -101,6 +101,7 @@ func newAuthenticateStateFromConfig(cfg *config.Config) (*authenticateState, err
 			Secure:   cfg.Options.CookieSecure,
 			HTTPOnly: cfg.Options.CookieHTTPOnly,
 			Expire:   cfg.Options.CookieExpire,
+			SameSite: cfg.Options.GetCookieSameSite(),
```

Comment:
just wondering if there might be a situation when this setting may need be different for authenticate endpoint and proxy.

Reply:
probably 🤷